example/spec/parallel/discoverGranules/DiscoverGranulesHttpsSpec.js

```diff
@@ -67,6 +68,12 @@ describe('The Discover Granules workflow with https Protocol', () => {
 
   afterAll(async () => {
     // clean up stack state added by test
+    await Promise.all(discoverGranulesLambdaOutput.payload.granules.map(
+      (granule) => granulesApiTestUtils.deleteGranule({
```

Comment:
Nit: This method is deprecated.  We shoudn't be using it, instead bring in the api-client granule delete method directly.